BSP/STM32F413/Src/BSP_Fans.c

```diff
@@ -137,7 +46,8 @@ int BSP_Fans_GetSpeed(uint8_t fan){
 ErrorStatus BSP_Fans_SetAll(uint32_t speed) {
     ErrorStatus result = SUCCESS;
     for (uint8_t i = 1; i <= 4; i++){
-        ErrorStatus e = BSP_Fans_Set(i, TOPSPEED);
+        if(i == CFAN) continue;
```

Comment:
Try following the styles guide (space between if and parentheses and braces for anything within, even if it is one line).